src/api/send/test.rs

```diff
@@ -217,7 +208,8 @@ fn missing_to_field() {
     assert_eq!(response.status(), Status::BadRequest);
 
     let body = response.body().unwrap().into_string().unwrap();
-    let error: AppError = AppErrorKind::MissingEmailParams(String::from("")).into();
+    let error: AppError =
+        AppErrorKind::InvalidPayload(String::from("missing field `to` at line 7 column 5")).into();
```

Comment:
Oh, small bit of collateral improvement here: previously we didn't have rich validation detail in these 400 errors, now we do. Winning!